snakemake/remote/SFTP.py

```diff
@@ -26,7 +26,13 @@ class RemoteProvider(AbstractRemoteProvider):
     allows_directories = True
 
     def __init__(
-        self, *args, keep_local=False, stay_on_remote=False, is_default=False, **kwargs
+        self,
+        *args,
+        keep_local=False,
+        stay_on_remote=False,
+        is_default=False,
+        mkdir_remote=False,
```

Comment:
I think it should be True by default then, right?

Reply:
I did not dare to set it to true in order not to break the existing code behaviour. But I agree, it would be better to set the default value to true. I'll change that.